snuba/consumers/schemas.py

```diff
@@ -4,6 +4,62 @@
 
 from snuba.utils.streams.topics import Topic
 
+_HARDCODED_SCHEMAS: Mapping[Topic, Mapping[str, Any]] = {
+    Topic.GENERIC_METRICS: {
+        "$schema": "http://json-schema.org/draft-06/schema#",
```

Comment:
This seems like quite an old version, should we try to use `2020-12` everywhere?